### PR TITLE
New Home page and other minor updates

### PR DESCRIPTION
**Pull request by rossfarrugia**
# Pull Request

- [x] This is a PR into production (main), and I am doing it from develop
- [x] If from develop, I have checked the code is working on the UAT site http://openpharma.s3-website.us-east-2.amazonaws.com/develop/

# Description
- Updated homepage with new hex wall and data viz for contributors (removed the contributor lists then at the bottom)
- Added hex logo package dev guidance
- Make PHUSE logo smaller due to navbar sizing impact
- Numerous minor package link fixes

## Metadata

Please reference any related issues here (using `#issuenumber`): #212 #239 #247
People to notify: @epijim @NNaikp

### CODE REVIEW

**Comment by NNaikp** [comment]
> @NNaikp see the update i made for datacutr and thevalidatoR - it didn't render in the test site hex wall. did i do anything wrong here?

@rossfarrugia when looking at the pipeline.yml then I've missed to add the execution of the new R script. I've rendered the site locally when doing updates and hence the rendered files have been committed. We should probably test it out in dev before merging.

**Comment by NNaikp** [comment]
Just pushed an update to the pipeline in dev

**Comment by NNaikp** [comment]
@rossfarrugia I think most/all issues with the environment is fixed now. 
Now we are hit by GH API's rate limit for the scraping being done for all contributors. This works locally as I am authenticated.
As far as I understand [this](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) we can utilise and automatically generated repo token.

I can test it out, but I don't have any experience in using it. 
Maybe @epijim has tried it before?

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ jobs:
         run: |
           Rscript scripts/add_badges.R
           Rscript scripts/add_contributions.R
+          Rscript scripts/add_hexwall_network.R
```
[the remark this answers is not in the document]
Comment (NNaikp) [reply .github/workflows/pipeline.yml:39]: Updated pipeline with this - thank you! 👍

**Comment by NNaikp** [comment]
Finally!

image


@rossfarrugia I just noticed the disclaimer is gone. Is that on purpose?

**Review by NNaikp — APPROVED** [review]
I have gone through the files and also checked that all badges are rendered as they should, so this pull requests should be good to go 🚀

**Comment by rossfarrugia** [comment]
Amazing effort @NNaikp - its now live on the site!! 🙌 you should post a message on announcements or general-chat channel in pharmaverse slack to let our community know. all credit to you and @epijim for this, thanks!!

**Pull request merged**